setup.py

```diff
@@ -25,7 +25,7 @@
         "coincurve>=10.0.0,<11.0.0",
         "eth-hash[pysha3];implementation_name=='cpython'",
         "eth-hash[pycryptodome];implementation_name=='pypy'",
-        "plyvel==1.0.5",
```

Comment:
I wonder if we should change this into a range to not break things for those that are stuck on something lower than `1.1.0` which is known to work well.

Reply:
Yeah, probably.  Just made that change.

Reply:
Ah wait, what I meant was to include everything from `1.0.5` up to whatever you say is working fine. Because `1.0.5` is the currently used version and I don't see a good reason to *require*  a higher version when `1.0.5` is known to work well. Especially because there may be people who can not easily upgrade to a higher version.

Reply:
As a reminder, `plyvel` is just the python interface library to LevelDB.  If we require a more recent `plyvel` version, it doesn't necessarily mean we're requiring a higher LevelDB version.  And it's a stronger guarantee that macOS users won't encounter the linked issue.

Also, `plyvel` has no sub-dependencies.  So that also makes it seem less risky to make this change.

What do you think?

Reply:
@cburgdorf By the way, with my comment above, you didn't mention being concerned about the level db version, but I figured I'd mention my thoughts on that just in case you were.

Reply:
Anyway, I'll just go with your suggestion.  I don't actually feel that strongly about it. 😄

Reply:
>As a reminder, plyvel is just the python interface library to LevelDB

Ah, that's a good point. That weakens my concerns. :+1: 

With that in mind, I guess a range of `plyvel>=1.1.0, <1.2.0` is fine.

>Anyway, I'll just go with your suggestion. I don't actually feel that strongly about it.

Sorry, for not making myself clear enough. I didn't mean to suggest a range without an upper bound such as `plyvel>=1.0.5"`.

Because I think that one can in fact become problematic. Current plyvel works under the assumption of having a `leveldb` of version `1.2` or higher on the system. An unbounded `plyvel` dependency might even install a `plyvel` version `2.x` which may depend on a `leveldb` `2.x` (hypothetical future versions) and with breaking changes things might fall apart then.

I'd pick one of these:

- `plyvel>=1.1.0, <2` (if we trust plyvel does semver right)
- `plyvel>=1.0.5, <2` (if we trust plyvel does semver right)
- `plyvel>=1.1.0, <1.2.0` (more conservative)
- `plyvel>=1.0.5, <1.2.0` (more conservative)

Reply:
I'll go with the last suggestion since it basically solves the macOS issue and also potentially avoids other issues with `plyvel` being incompatible with certain LevelDB versions.